app/Http/Requests/Devices/DeviceStoreRequest.php

```diff
@@ -14,9 +14,10 @@ class DeviceStoreRequest extends Request
     public function rules()
     {
         return [
-            'name' => 'required|max:45',
-            'alias' => 'required|max:45',
-            'token' => 'required|max:20',
+            'name' => 'max:45',
+            'uuid' => '',
```

Comment:
ik geloof dat dit garandeerd dat het veld altijd zal bestaan, zelfs als de gebruiker het niet meegeeft. Helpt bij nullpointer exceptions in de repository, maar ik heb op dat niveau al iets anders gedaan, dus is dit legacy. Zal ik zo ff controleren